src/main/java/net/syscon/elite/api/model/ImprisonmentStatus.java

```diff
@@ -0,0 +1,20 @@
+package net.syscon.elite.api.model;
+
+import lombok.Data;
+
+@Data
+public class ImprisonmentStatus {
+
+    private Long bookingId;
+    private String bandCode;
+    private String imprisonmentStatus;
+    private String legalStatus;
+
+    public void deriveLegalStatus() {
+        if (this.bandCode != null) {
+            final var bandCode = Integer.parseInt(this.bandCode);
+            this.legalStatus = (bandCode <= 8 || bandCode == 11) ? "Convicted" : "Remand";
```

Comment:
Not sure if you want to enhance this now, but I think months ago we talked about deportation status that might be recorded incorrectly as  Convicted. Probably something for the future

Reply:
Indeed